hack/preview.sh

```diff
@@ -39,12 +39,30 @@ if [ -n "$MY_GITHUB_ORG" ]; then
     $ROOT/hack/util-set-github-org $MY_GITHUB_ORG
 fi
 
+
+echo
+echo -n "Waiting for 'spi-system' namespace to exist: "
+while ! kubectl get namespace spi-system --kubeconfig ${KCP_KUBECONFIG} &> /dev/null ; do
```

Comment:
argoCD is not applied yet, so the spi-system namespace will not be created. The current code is creating the namespace and route, which is then updated by argoCD